source/localizable/pricing.html.erb

```diff
@@ -17,53 +17,66 @@ header_class: header-hero
   </div>
 </div>
 
-<section class="princing-plans">
+<section class="princing-plans border-bottom">
   <div class="row plans">
 
     <div id="plan-free" class="plan">
-      <h3><%= t("pricing.plans.free.title") %></h3>
-      <div class="plan-price">
-        <span class="currency">&euro;</span>0
+      <div class="plan-content">
+        <h3><%= t("pricing.plans.free.title") %></h3>
+        <div class="plan-price">
+          <span class="currency">&euro;</span>0
+        </div>
+        <% pricing_features = t("pricing.plans.free.features")
+        pricing_features = Array.new unless pricing_features.is_a?(Array)
+        pricing_features.each do |feature| %>
+          <%= markdown feature %>
```

Comment:
maybe it's nice to have a helper for this, where you give up just the name of the plan.